ordered_product/tests/test_edit_ordered_product.py

```diff
@@ -0,0 +1,80 @@
+import pytest
+from ordered_product.models import OrderedProduct
+from client.models import Client
+from django.core.exceptions import ObjectDoesNotExist
+
+
+def client_check(user):
+    try:
+        return Client.objects.get(client_account=user)
+    except ObjectDoesNotExist:
+        return False
+
+
+class TestsEditOrderedProduct:
+    @pytest.mark.django_db()
+    def test_change_valid_quantity_to_valid_order(self, client, saved_client0, ordered_product0, client0_login):
```

Comment:
This test looks like a classic case of pytest.mark.parametrize
read more here: https://docs.pytest.org/en/6.2.x/parametrize.html

Reply:
@MeitarRizner I agree. A test case should be about one case. You can parametrize it to cover multiple scenarios, or if you don't have the time to learn that stuff before the deadline, you need to split it into multiple cases.

Reply:
I did it in the lase update